tests/environment/docker-compose-legacy-transforms-only.yml

```diff
@@ -24,28 +24,22 @@ services:
                 -Ddb.url=jdbc:postgresql://postgres:5432/alfresco
                 -Dalfresco.restApi.basicAuthScheme=true
                 -Dimap.server.enabled=true
+                -Dimap.server.port=1143
```

Comment:
do you need imap, ftp and etc for transform tests? If not it is better to remove those properties

Reply:
Removed imap and ftp.config and test pass